WS2012R2/lisa/xml/Kdump_Tests.xml

```diff
@@ -54,11 +54,12 @@
         <test>
             <testName>Crash_single_core</testName>
             <setupScript>
+                <file>setupscripts\RevertSnapshot.ps1</file>
                 <file>setupscripts\ChangeCPU.ps1</file>
                 <file>setupScripts\SetVMMemory.ps1</file>
             </setupScript>
             <testScript>setupScripts\kdump.ps1</testScript>
-            <files>remote-scripts/ica/utils.sh,remote-scripts/ica/kdump_config.sh,remote-scripts/ica/kdump_results.sh,remote-scripts/ica/kdump_execute.sh</files>
+            <files>remote-scripts/ica/utils.sh,remote-scripts/ica/kdump_config.sh,remote-scripts/ica/kdump_results.sh,remote-scripts/ica/kdump_execute.sh,remote-scripts/ica/kdump_nfs_config.sh</files>
```

Comment:
Yes, after submitted pull requests, i noted the mistake.
Just left it for Crash_nfs case